assets/src/dashboard/utils/usePagePreviewSize.js

```diff
@@ -16,55 +16,52 @@
 /**
  * External dependencies
  */
-import { useCallback, useEffect, useState } from 'react';
+import { useEffect, useMemo, useState } from 'react';
 /**
  * Internal dependencies
  */
-import theme from '../theme';
 import { PAGE_RATIO } from '../constants';
+import theme from '../theme';
+
+const THUMBNAIL_WIDTH = 33;
```

Comment:
This is a theme variable `theme.previewWidth.thumbnail`

Reply:
will check rq. when I started on this file it was just the number

Reply:
Oh there is! good call

Reply:
updated.